modules/core/src/main/java/com/google/refine/importers/ImporterUtilities.java

```diff
@@ -198,6 +198,26 @@ static public void setupColumns(Project project, List<String> columnNames) {
         project.columnModel.update();
     }
 
+    /**
+     * @param columnsHasData
+     *            Record if there is data in each column( false:null;true:has data)
+     */
+    static public void deleteEmptyColumns(Project project, List<Boolean> columnsHasData) throws ModelException {
```

Comment:
```suggestion
    static void deleteEmptyColumns(Project project, List<Boolean> columnsHasData) throws ModelException {
```

Since this is mainly an internal helper, let's not make it public. Given that, I don't think it makes a huge difference what it takes for parameters (although I'd suggest boolean[] for efficiency) instead of List<Boolean>), for reasons you'll see in my comments on the caller.

Reply:
removed the public modifier